[33mWARNING [0m selenium.webdriver.common.selenium_manager:selenium_manager.py:138 The chromedriver version (130.0.6723.69) detected in PATH at C:\WebDriverbin\bin\chromedriver.exe might not be compatible with the detected chrome version (131.0.6778.265); currently, chromedriver 131.0.6778.264 is recommended for chrome 131.*, so it is advised to delete the driver in PATH and retry
[33mWARNING [0m urllib3.connectionpool:connectionpool.py:870 Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000002553CD66A50>: Failed to establish a new connection: [WinError 10061] –ü–æ–¥–∫–ª—é—á–µ–Ω–∏–µ –Ω–µ —É—Å—Ç–∞–Ω–æ–≤–ª–µ–Ω–æ, —Ç.–∫. –∫–æ–Ω–µ—á–Ω—ã–π –∫–æ–º–ø—å—é—Ç–µ—Ä –æ—Ç–≤–µ—Ä–≥ –∑–∞–ø—Ä–æ—Å –Ω–∞ –ø–æ–¥–∫–ª—é—á–µ–Ω–∏–µ')': /session/ad34bb8d65ff32fd427f74234f6c391a
[33mWARNING [0m urllib3.connectionpool:connectionpool.py:870 Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000002553C8AA7B0>: Failed to establish a new connection: [WinError 10061] –ü–æ–¥–∫–ª—é—á–µ–Ω–∏–µ –Ω–µ —É—Å—Ç–∞–Ω–æ–≤–ª–µ–Ω–æ, —Ç.–∫. –∫–æ–Ω–µ—á–Ω—ã–π –∫–æ–º–ø—å—é—Ç–µ—Ä –æ—Ç–≤–µ—Ä–≥ –∑–∞–ø—Ä–æ—Å –Ω–∞ –ø–æ–¥–∫–ª—é—á–µ–Ω–∏–µ')': /session/ad34bb8d65ff32fd427f74234f6c391a
[33mWARNING [0m urllib3.connectionpool:connectionpool.py:870 Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x0000025539D32900>: Failed to establish a new connection: [WinError 10061] –ü–æ–¥–∫–ª—é—á–µ–Ω–∏–µ –Ω–µ —É—Å—Ç–∞–Ω–æ–≤–ª–µ–Ω–æ, —Ç.–∫. –∫–æ–Ω–µ—á–Ω—ã–π –∫–æ–º–ø—å—é—Ç–µ—Ä –æ—Ç–≤–µ—Ä–≥ –∑–∞–ø—Ä–æ—Å –Ω–∞ –ø–æ–¥–∫–ª—é—á–µ–Ω–∏–µ')': /session/ad34bb8d65ff32fd427f74234f6c391a